### PR TITLE
feat(Labels): Redesign labels on mobile & web

### PR DESCRIPTION
**Pull request by vp8x8**
header
scaled

### CODE REVIEW

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL.

**Comment by saghul** [comment]
Screenshot 2021-04-06 at 16 00 18

We need the text to be elided when too long.

File: ios/Podfile.lock [hunk 1]
```diff
@@ -552,13 +552,13 @@ SPEC CHECKSUMS:
   React-jsi: ddb471a56185e4007835a1bba0882ecb5ce3dc0e
   React-jsiexecutor: 67106691c60030ec888d7cbbc4f48a3168e27a02
   React-jsinspector: 92ceee6c66dc19886289b52436ade7e020b89602
-  react-native-background-timer: e0384ea2fa5a98f67f84f9c4dc274260ddd674ed
-  react-native-calendar-events: 1442fad71a00388f933cfa25512588fec300fcf8
-  react-native-keep-awake: eba3137546b10003361b37c761f6c429b59814ae
-  react-native-netinfo: 8d8db463bcc5db66a8ac5c48a7d86beb3b92f61a
+  react-native-background-timer: 029c606b3fd6dd476b153e177c518b6ade4c169f
```
[the remark this answers is not in the document]
Comment (vp8x8) [reply ios/Podfile.lock]: Removed.

File: react/features/conference/components/web/Labels.js [hunk 2]
```diff
@@ -4,116 +4,61 @@ import React from 'react';
...
+ * @param {Object} state - The redux state.
+ * @private
+ * @returns {{
+ *     _filmstripVisible: boolean,
+ *     _showVideoQualityLabel: boolean
+ * }}
+ */
+function _mapStateToProps(state: Object) {
+    return {
+        _showVideoQualityLabel: !(shouldDisplayTileView(state) || interfaceConfig.VIDEO_QUALITY_LABEL_DISABLED)
```
[the remark this answers is not in the document]
Comment (vp8x8) [reply react/features/conference/components/web/Labels.js]: Done.

[the remark this answers is not in the document]
Comment (saghul) [reply react/features/conference/components/web/Labels.js]: Wonderful!

**Review by saghul — APPROVED** [review]
Excellent work Vlad! 👏

**Pull request merged**